en/intro/conventions.rst

```diff
@@ -142,6 +142,18 @@ Now that you've been introduced to CakePHP's fundamentals, you might try a run
 through the :doc:`/tutorials-and-examples/bookmarks/intro` to see how things fit
 together.
 
+ORM and FormHelper Naming Conventions
+=====================================
+
+While navigating the CakePHP source code, when dealing with the database schema,
+fields are often referred to as columns but when dealing with entities,
+they are often referred to as properties.
+
+Fields/columns described as keys usually reflect primary or foreign key columns.
+
+When looking for schema columns, database fields and entity properties, make
```

Comment:
This sentence is bit of an overkill.
Especially since this is an intro  (!) text and probably one of the first pages a CakePHP noob reads 😆 

![conventsions](https://cloud.githubusercontent.com/assets/625761/23557702/31d22e40-0031-11e7-9ac3-6363f2b203e4.png)